### ISSUE TITLE
[ACCESSIBILITY] Add alt text field for image uploads

### ISSUE THREAD

**Issue by vburgett** [rokwire/illinois-app#4836]
When uploading an image for an event, a group, or anywhere else in the app or on the app website, a user should be required to enter alt text or use a checkbox to bypass the alt text field. 

Alt Text: Text entry field
_Add a brief one to two sentence text description of the image._

Checkbox: This image is decorative and does not convey any information. Note: logos, departmental banners, and images used as links are all considered informational.

This is a solution for ticket #1964. cc: @acovher2

**Comment by TodorBachvarov** [comment]
Hi @vburgett, I've created a preview build (7.3.8-alter-text-preview-dev) so you can see the UI part progress and share your touts. Since we want to add this new field for every image we upload from the App I've added the UI into ImageEditPanel and the dialog from where we can add image by URL: 

Example Group -> create post -> add image
![Image](https://github.com/user-attachments/assets/3a1ede8b-814f-455a-b03f-f18ea67c8bf4)

If use URL
![Image](https://github.com/user-attachments/assets/2cccac3c-b94d-4b0e-aa6c-ac2050bd10b4)

if choose from device
![Image](https://github.com/user-attachments/assets/cc00d748-019a-4bf2-8c86-79675436beb0)

**Comment by acovher2** [comment]
@vburgett @TodorBachvarov  
The placeholder changes are verified as passing.

This field is still not containing an asterisk indicating *Required field and it is savable without entering. The Alt text has to be a **required** field in all areas where image upload is possible. Currently, in 7.3.26 the Profile image can be added without, so can group image and I suspect all image upload areas.

Once the field is required, future uploads will require alt text, which will resolve the forward-facing issue.

For existing images, we need a remediation plan:
(1) Export a list of all images without alt text;
(2) Provide a way for content owners to add alt text retroactively;
(3) Prioritize critical images; and
(4) Validate updates with accessibility QA.
This ensures full compliance with WCAG and avoids an accessibility gap.

**Comment by acovher2** [comment]
Verified as fixed in 7.3.41
